src/getViewForSwagger2.ts

```diff
@@ -76,6 +77,65 @@ export function getViewForSwagger2(opts: CodeGenOptions): ViewData {
   };
 }
 
+function normalizeResponseDefinitions(swagger: any): any {
```

Comment:
We can probably use `CodeGenOptions['swagger']` here and for the return type.

Reply:
@Kosta-Github I think once we adjust this type and resolve the conflict we can merge this. Since you mentioned you do not have so much time now, if you could resolve the conflict I can fix this up separately.